core/services/relay/evm/config_poller.go

```diff
@@ -3,244 +3,111 @@ package evm
-	}, nil
-}
-
-func unpackLogData(d []byte, withFeedId bool) ([]interface{}, error) {
-	args := makeConfigSetMsgArgs(withFeedId)
-	unpacked, err := args.Unpack(d)
-	if err != nil {
-		return nil, err
-	}
-	if len(unpacked) != len(args) {
-		return nil, errors.Errorf("invalid number of fields, got %v", len(unpacked))
-	}
-	return unpacked, nil
-}
-
-func ConfigFromLog(logData []byte) (FullConfigFromLog, error) {
-	unpacked, err := unpackLogData(logData, false)
-	if err != nil {
-		return FullConfigFromLog{}, err
-	}
-	contractConfig, err := NewContractConfigFromLog(unpacked, firstIndexWithoutFeedId)
-	if err != nil {
-		return FullConfigFromLog{}, err
-	}
-	return FullConfigFromLog{
-		FeedId:         [32]byte{},
-		ContractConfig: contractConfig,
-	}, nil
-}
-
-func ConfigFromLogWithFeedId(logData []byte) (FullConfigFromLog, error) {
-	unpacked, err := unpackLogData(logData, true)
-	if err != nil {
-		return FullConfigFromLog{}, err
-	}
-	feedId, ok := unpacked[0].([32]byte)
-	if !ok {
-		return FullConfigFromLog{}, errors.Errorf("invalid feed ID, got %T", unpacked[0])
-	}
-	contractConfig, err := NewContractConfigFromLog(unpacked, firstIndexWithFeedId)
-	if err != nil {
-		return FullConfigFromLog{}, err
-	}
-
-	return FullConfigFromLog{
-		FeedId:         feedId,
-		ContractConfig: contractConfig,
+		F:                     unpacked.F,
+		OnchainConfig:         unpacked.OnchainConfig,
+		OffchainConfigVersion: unpacked.OffchainConfigVersion,
+		OffchainConfig:        unpacked.OffchainConfig,
 	}, nil
 }
 
-type ConfigPoller struct {
+type configPoller struct {
 	lggr               logger.Logger
 	filterName         string
 	destChainLogPoller logpoller.LogPoller
 	addr               common.Address
-	feedId             common.Hash
-}
-
-type ConfigPollerOption func(cp *ConfigPoller)
-
-func WithFeedId(feedId common.Hash) ConfigPollerOption {
-	return func(cp *ConfigPoller) {
-		cp.feedId = feedId
-	}
 }
 
-func NewConfigPoller(lggr logger.Logger, destChainPoller logpoller.LogPoller, addr common.Address, opts ...ConfigPollerOption) (*ConfigPoller, error) {
+// NewConfigPoller creates a new ConfigPoller
+func NewConfigPoller(lggr logger.Logger, destChainPoller logpoller.LogPoller, addr common.Address) (ConfigPoller, error) {
 	configFilterName := logpoller.FilterName("OCR2ConfigPoller", addr.String())
+
 	err := destChainPoller.RegisterFilter(logpoller.Filter{Name: configFilterName, EventSigs: []common.Hash{ConfigSet}, Addresses: []common.Address{addr}})
 	if err != nil {
 		return nil, err
 	}
 
-	cp := &ConfigPoller{
+	cp := &configPoller{
 		lggr:               lggr,
 		filterName:         configFilterName,
 		destChainLogPoller: destChainPoller,
 		addr:               addr,
 	}
 
-	for _, opt := range opts {
-		opt(cp)
-	}
-
 	return cp, nil
 }
 
-func (lp *ConfigPoller) WithFeedId() bool {
-	return lp.feedId != (common.Hash{})
+// Notify noop method
+func (lp *configPoller) Notify() <-chan struct{} {
+	return nil
```

Comment:
Why are these `lp`, could we change them to `cp` while we're at it?  Possible confusion with logpoller